src/printer.js

```diff
@@ -654,7 +654,7 @@ function genericPrintNoParens(path, options, print) {
       var isTypeAnnotation = n.type === "ObjectTypeAnnotation";
       var isTypeScriptTypeAnnotaion = n.type === "TSTypeLiteral";
       // Leave this here because we *might* want to make this
-      // configurable later -- flow accepts ";" for type separators, 
```

Comment:
Trailing space removal from my Emacs configuration, sorry if it bloats the diff.